src/Magnum/BulletIntegration/Test/IntegrationTest.cpp

```diff
@@ -106,6 +123,46 @@ void IntegrationTest::quaternion() {
     CORRADE_VERIFY(btQuaternion{a} == b);
 }
 
+void IntegrationTest::motionState() {
+    /* Setup Bullet Physics */
+    btDefaultCollisionConfiguration collisionConfig;
+    btCollisionDispatcher dispatcher{&collisionConfig};
+    btDbvtBroadphase broadphase;
+    btDiscreteDynamicsWorld btWorld{&dispatcher, &broadphase, nullptr, &collisionConfig};
+    btWorld.setGravity(btVector3{0.0f, 0.0f, 0.0f});
+
+    /* Setup scene graph */
+    Scene3D scene;
+    Object3D parent{&scene};
+    Object3D child{&parent};
+
+    parent.translate(Vector3{1.0f, 2.0f, 3.0f})
+          .rotateX(45.0_degf)
+          .rotateY(25.0_degf)
+          .rotateZ(15.0_degf);
+    child.translate(Vector3{30.0f, 20.0f, 10.0f})
+          .rotateZ(32.0_degf)
+          .rotateY(22.0_degf)
+          .rotateX(12.0_degf);
+
+    /* Setup rigid body which will receive the motion state */
+    MotionState motionState{child};
+    btSphereShape collisionShape{0.0};
```

Comment:
This should be `0.0f` -- some compilers are taking type narrowing very seriously. I think you should get at least a warning from the compiler.

Btw., how is it with `btScalar`? Is it always `float` or can it be switched to `double` at compile time?